diff --git a/apps/openmw/mwgui/settingswindow.cpp b/apps/openmw/mwgui/settingswindow.cpp
index 5397a31..dc4113f 100644
--- a/apps/openmw/mwgui/settingswindow.cpp
+++ b/apps/openmw/mwgui/settingswindow.cpp
@@ -185,7 +185,7 @@ namespace
         { "off", "balance", "performance", "aggressive" };
 
     constexpr float landOptimizedDistanceMin = 4000.f;
-    constexpr float landOptimizedDistanceMax = 40000.f;
+    constexpr float landOptimizedDistanceMax = 40960.f;
     void clampOptimizedLandDistance(float distance)
     {
         distance = std::clamp(distance, landOptimizedDistanceMin, landOptimizedDistanceMax);
diff --git a/apps/openmw/mwrender/renderingmanager.cpp b/apps/openmw/mwrender/renderingmanager.cpp
index e668d76..44d0345 100644
--- a/apps/openmw/mwrender/renderingmanager.cpp
+++ b/apps/openmw/mwrender/renderingmanager.cpp
@@ -955,7 +955,7 @@ namespace MWRender
         Nif::NIFFile::setLoadUnsupportedFiles(Settings::Manager::getBool("load unsupported nif files", "Models"));
 
         mNearClip = Settings::Manager::getFloat("near clip", "Camera");
-        mConfiguredViewDistance = Settings::Manager::getFloat("viewing distance", "Camera");
+        mConfiguredViewDistance = std::clamp(Settings::Manager::getFloat("viewing distance", "Camera"), 2500.f, 40960.f);
         mViewDistance = mConfiguredViewDistance;
         mLandOptimizationDistance = mConfiguredViewDistance;
         updateLandOptimizationProfile();
@@ -2015,7 +2015,7 @@ namespace MWRender
             }
             else if (setting.first == "Camera" && setting.second == "viewing distance")
             {
-                mConfiguredViewDistance = Settings::Manager::getFloat("viewing distance", "Camera");
+                mConfiguredViewDistance = std::clamp(Settings::Manager::getFloat("viewing distance", "Camera"), 2500.f, 40960.f);
                 updateLandOptimizationProfile();
                 mLandOptimizationDistance = mConfiguredViewDistance;
                 resetLandOptimization(false);
diff --git a/files/mygui/openmw_settings_window.layout b/files/mygui/openmw_settings_window.layout
index d93322b..4233340 100644
--- a/files/mygui/openmw_settings_window.layout
+++ b/files/mygui/openmw_settings_window.layout
@@ -495,14 +495,14 @@
               <Property key="Caption" value="#{sRender_Distance}"/>
             </Widget>
             <Widget type="ScrollBar" skin="MW_HScroll" position="0 28 462 16" name="LargeRenderingDistanceSlider">
-              <Property key="Range" value="5693"/>
-              <Property key="Page" value="256"/>
+              <Property key="Range" value="38461"/>
+              <Property key="Page" value="512"/>
               <UserString key="SettingType" value="Slider"/>
               <UserString key="SettingCategory" value="Camera"/>
               <UserString key="SettingName" value="viewing distance"/>
               <UserString key="SettingValueType" value="Cell"/>
               <UserString key="SettingMin" value="2500"/>
-              <UserString key="SettingMax" value="8192"/>
+              <UserString key="SettingMax" value="40960"/>
               <UserString key="SettingLabelWidget" value="RenderDistanceLabel"/>
               <UserString key="SettingLabelCaption" value="#{sRender_Distance} (x%s)"/>
             </Widget>
@@ -522,14 +522,14 @@
               <UserString key="Caption_Text" value="#{arenamp=settings.tooltip.optimized_distance}"/>
             </Widget>
             <Widget type="ScrollBar" skin="MW_HScroll" position="0 28 462 16" name="OptimizedRenderingDistanceSlider">
-              <Property key="Range" value="36001"/>
+              <Property key="Range" value="36961"/>
               <Property key="Page" value="100"/>
               <UserString key="SettingType" value="Slider"/>
               <UserString key="SettingCategory" value="Camera"/>
               <UserString key="SettingName" value="viewing distance"/>
               <UserString key="SettingValueType" value="Integer"/>
               <UserString key="SettingMin" value="4000"/>
-              <UserString key="SettingMax" value="40000"/>
+              <UserString key="SettingMax" value="40960"/>
               <UserString key="SettingLabelWidget" value="OptimizedRenderDistanceLabel"/>
               <UserString key="SettingLabelCaption" value="#{arenamp=settings.optimized_distance_value}"/>
             </Widget>
@@ -537,7 +537,7 @@
               <Property key="Caption" value="4 000"/>
             </Widget>
             <Widget type="TextBox" skin="SandText" position="362 46 100 16">
-              <Property key="Caption" value="40 000"/>
+              <Property key="Caption" value="40 960"/>
               <Property key="TextAlign" value="Right"/>
             </Widget>
           </Widget>
diff --git a/files/ui/graphicspage.ui b/files/ui/graphicspage.ui
index df7cdbb..66816fc 100644
--- a/files/ui/graphicspage.ui
+++ b/files/ui/graphicspage.ui
@@ -516,7 +516,7 @@
              <number>512</number>
             </property>
             <property name="maximum">
-             <number>81920</number>
+             <number>40960</number>
             </property>
             <property name="value">
              <number>8192</number>
